src/components/tool/guide/Config.js

```diff
@@ -21,6 +21,8 @@ export const DEFAULT_CONFIG = {
   ram: 16,
   gpuModules: 1,
   wallTime: { hour: 1, minute: 0 },
+  isArrayJob: "No",
+  arrayJobConfig: { firstIndex: 1, upperBound: 1, stepper: 1 },
```

Comment:
Having a default value that is not valid is a bit confusing. Maybe have 1-10 as the default

Reply:
Updated